parser.go

```diff
@@ -775,3 +775,7 @@ func ParseRule(rule string) (*Rule, error) {
 
 	return r, nil
 }
+
+func ParseRule(rule string) (*Rule, error) {
```

Comment:
all exported functions should have comments of the form
`// ParseRule blah blah.`

Reply:
This should probably just be the existing comment on `parseRuleAux()`